tests/test_eas_csc.py

```diff
@@ -31,6 +32,22 @@
 
 
 class CscTestCase(salobj.BaseCscTestCase, unittest.IsolatedAsyncioTestCase):
+    @contextlib.asynccontextmanager
```

Comment:
I would rather you add this in a separate ticket and really implement unit tests..